cmd/run_scenario.go

```diff
@@ -107,7 +108,18 @@ func (r *scenarioRunner) run(ctx context.Context) error {
 		if len(pieces) != 2 {
 			return fmt.Errorf("option does not have '='")
 		}
-		scenarioOptions[pieces[0]] = pieces[1]
+		key, value := pieces[0], pieces[1]
+
+		if strings.HasPrefix(value, "@") {
```

Comment:
This needs to get documented somewhere... I'm not sure if it makes sense to do on every option that can take JSON or if there's some general place that would make more sense, but, we should have it mentioned somewhere.

Reply:
I've added it to the README in the context of the throughput_stress scenario now.